packages/salesforcedx-vscode-replay-debugger/src/breakpoints/breakpointUtil.ts

```diff
@@ -36,6 +50,36 @@ export class BreakpointUtil {
     );
   }
 
+  public getRawLineBPInfo(): any | undefined {
+    return this.rawLineBPInfo;
+  }
+
+  public createMappingsFromLineBreakpointInfo(lineBpInfo: any): void {
+    this.rawLineBPInfo = lineBpInfo;
+    for (const info of lineBpInfo) {
+      if (!this.lineNumberMapping.has(info.uri)) {
+        this.lineNumberMapping.set(info.uri, []);
+      }
+      this.lineNumberMapping.set(
+        info.uri,
+        this.lineNumberMapping.get(info.uri)!.concat(info.lines)
+      );
+      this.typerefMapping.set(info.typeref, info.uri);
+    }
+  }
+
+  public returnLinesForLoggingFromBreakpointArgs(
```

Comment:
This iterates through each object, grabs lines numbers, and joins them all with commas. Can this be simplified to `bpArr.map(bp => bp.line).join(‘,’)`?

Reply:
Yes, done. It'll be updated with the next iteration.